src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -45,6 +45,7 @@ const (
 	// zero means -> bring all logs
 	controllerLogsSecondsAgo  = 0
 	consoleOperatorName       = "console"
+	consoleCapabilityName     = "Console"
```

Comment:
Can we:
1. Stop our weird `go.mod` `replace` directive for openshift/api and bring it as a proper dependency?
2. Use [the actual constant](https://github.com/openshift/api/blob/7287964c76316b1d6636d31d7df7f82ee1e33d45/config/v1/types_cluster_version.go#L251) for this

Let's not hold back this PR though and follow up with a separate PR

Reply:
This would be the first step, I think, just updating to Go 1.19: #575.